_pages/en/capabilities.md

```diff
@@ -4,47 +4,218 @@ title: High Performing IT Capabilities
 ref: it-capabilities
 lang: en
 status: posted
-sections: Archives
+sections: Work In Progress
 permalink: /high-performing-it-capabilities.html
 ---
 
 ## High Performing IT Capabilities
```

Comment:
should reword to "High Performing DevOps Capabilities"

There are other IT capabilities that we are not touching here (like asset management, demand management, supply management, sourcing and procurement management).